WordPress/twentytwentyone
# Remove horizontal margins from post titles in post pagination

Fixes #438 

## Summary

Post title underline is off-centered on hover in post navigation component.
See animated screenshot below:
![79016d2d8f675321271d9f887a712d0d](https://user-images.githubusercontent.com/1590998/95798933-81d8fe00-0cf3-11eb-8618-63a73d6697ae.gif)

## Relevant technical choices:
I think we don't really need any margin. It looks really great when the post title is aligned with the arrow.
…and it fixes the issue 😏
![a77c78eddf2cf1753ed8eb45031650b5](https://user-images.githubusercontent.com/1590998/95798977-96b59180-0cf3-11eb-8eb3-5303cb82fc0c.gif)

## Test instructions

This PR can be tested by following these steps:
1. Go to the post navigation area in a single post (you'll need several posts to see the issue in action)
2. Hover the post title for previous or next post
3. Post title is not off-centered anymore! 🎉 

## Quality assurance
